website/content/en/docs/amazon-sagemaker-integration/sagemaker-components-for-kubeflow-pipelines.md

```diff
@@ -14,6 +14,40 @@ You can create a Kubeflow Pipeline built entirely using SageMaker components, or
 
 There are two versions of SageMaker components - boto3 based v1 components and SageMaker Operator for K8s (ACK) based v2 components. 
 
+
+## Configure Permissions for Pipeline pods to access SageMaker
+
+
+### Configuration for SageMaker Components V2
+
+There is no additional configuration required for SageMaker Components V2 if you have already installed the AWS distribution of Kubeflow.
+
+### Configuration for SageMaker Components V1
+
+If you are also using the SageMaker components version 1. Grant SageMaker access to the service account used by kubeflow pipeline pods. Otherwise you can skip this section.
```

Comment:
I think we should start with Grant SageMaker access

To use SageMaker Components version 1, grant SageMaker access to the service account used by Kubeflow pipeline pods. We recommend completing these steps to avoid configuration in future but you can skip this section if you do not intend to use version 1 of the components: